waypoints:
  pre_table1:
  - {x: 6.927, y: -27.866, theta: 1.757, frame: "map"}
  table1:
  - {x: 6.650, y: -27.252, theta: 1.834, frame: "map"}
  post_table1:
  - {x: 7.000, y: -28.636, theta: -1.315, frame: "map"}
  pre_table2:
  - {x: 6.927, y: -27.866, theta: 1.757, frame: "map"}
  table2:
  - {x: 6.650, y: -27.252, theta: 1.834, frame: "map"}
  post_table2:
  - {x: 7.000, y: -28.636, theta: -1.315, frame: "map"}

object_constraints:
  cube:
    bounds: { xmin: 0.034, xmax: 0.12, ymin: 0.034, ymax: 0.12, zmin: 0.034, zmax: 0.12 }
    location: { xmin: -.inf, xmax: .inf, ymin: -.inf, ymax: .inf, zmin: 0.5, zmax: .inf }

arm_gripper_poses:
  drop:
    frame: "base_link"
    position: { x: 0.7, y: 0.0, z: 1.0 }
    orientation: { x: 0, y: 0, z: 0, w: 1.0 }

arm_joint_poses:
  tuck: [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
  ready: [1.55, -0.91, 0.98, -1.18, 0.14, -1.06, 0.0]
  stow: [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  drop: [0.54, 0.06, 1.31, -1.15, 1.61, -1.68, 0.0]

  ready_for_candy: [1.39, -0.71, 1.56, -1.40, 0.41, -1.42, 0.69]
  pre_pick: [1.42, 0.50, 1.54, -1.83, 2.13, -1.63, 0.12]
  pick: [1.58, 0.98, 1.68, -1.34, 2.52, -1.15, 0.12]
  candy_verify_1: [1.47, 0.88, 1.80, -1.67, 0.91, -1.31, -1.53]
  candy_verify_2: [1.47, 0.88, 1.80, -1.67, 0.91, -1.31, 0.11]
  post_pick: [1.50, -0.34, 1.17, -1.42, 1.28, -1.95, -0.07]

  hat_reach_start: [1.60, -0.42, 2.72, 0.79, 0.27, 2.08, 1.38]

trajectories:
  tuck_to_stow:
  - [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 0.82, 0.00]
  - [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  stow_to_tuck:
  - [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 0.82, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
